src/openrpc/media_info.json

```diff
@@ -0,0 +1,425 @@
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/PipelineId"
+                    }
+                }
+            ],
+            "result": {
+                "name": "audioFormat",
+                "summary": "Audio format used by the pipeline.",
+                "schema": {
+                    "$ref": "#/components/schemas/AudioFormat"
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [
+                        {
+                            "name": "pipelineId",
+                            "value": 1
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": {
+                            "bitRate": 128,
+                            "channels": "2",
+                            "codec": "aac",
+                            "container": "audio/mp4",
+                            "pipelineId": 1,
+                            "sampleRate": 48
+                        }
+                    }
+                }
+            ]
+        },
+        {
+            "name": "onActiveAudioFormatsChanged",
+            "summary": "Get notified when the active audio formats used across all media pipelines starts, stops, or changes.",
+            "tags": [
+                {
+                    "name": "event"
+                },
+                {
+                    "name": "capabilities",
+                    "x-manages": [
+                        "xrn:firebolt:capability:media-info:audio-format"
+                    ]
+                }
+            ],
+            "params": [],
+            "result": {
+                "name": "onActiveAudioFormatsChanged",
+                "summary": "The updated audio formats used across all pipelines.",
+                "schema": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/components/schemas/AudioFormat"
+                    }
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [],
+                    "result": {
+                        "name": "Default Result",
+                        "value": [
+                            {
+                                "bitRate": 128,
+                                "channels": "5.1",
+                                "codec": "aac",
+                                "container": "audio/mp4",
+                                "pipelineId": 1,
+                                "sampleRate": 48
+                            }
+                        ]
+                    }
+                }
+            ]
+        },
+        {
+            "name": "onActiveVideoFormatsChanged",
```

Comment:
We don't need this method explicitly defined because activeVidioFormats has been tagged with _**property**_